### ISSUE TITLE
write an explanation on how to get an api key & an authorized access token to feed to this gem

### ISSUE THREAD

**Issue by dezmathio** [dezmathio/underarmour#11]

**Comment by franzejr** [comment]
Suppose someone would like to use this Gem. This person should get the token, registering an app. Once this is done, you can use the `API_TOKEN` as an environment variable.

This is your idea, @dezmathio?

**Comment by dezmathio** [comment]
yup, exactly. I have a little local sinatra app that I've been using, I will make it open source so people can just easily set it up. Just focusing on this right now :)
